test/framework/releaseVersions.go

```diff
@@ -82,6 +72,39 @@ func (e *ClusterE2ETest) GetLatestMinorReleaseBinaryFromReleaseBranch(releaseBra
 	return binaryPath
 }
 
+func (e *ClusterE2ETest) GetReleaseFromVersion(version *semver.Version) string {
+	releases := e.prodReleases()
+
+	var targetVersion *releasev1alpha1.EksARelease
+	for _, release := range releases.Spec.Releases {
+		releaseVersion, err := semver.New(release.Version)
+		if err != nil {
+			e.T.Fatal(err)
+		}
+		if releaseVersion == version {
+			targetVersion = &release
+		}
+	}
+
+	binaryPath, err := e.getBinary(targetVersion)
+	if err != nil {
+		v := fmt.Sprintf("v%d.%d.%d", version.Minor, version.Minor, version.Patch)
+		e.T.Fatalf("Failed getting binary for specified version %s: %s", v, err)
+	}
+
+	return binaryPath
+}
+
+func (e *ClusterE2ETest) GetReleaseVersion() string {
```

Comment:
What's the intended use case for this function?

Reply:
I was playing around with the function to set the version to the latest; was going back and forth between two approaches:
- get the latest from the release in the same way we get each other release
- just use a static path (right now, `anywhere`), as the latest/default

This method is hanging out until an approach is settled on

Reply:
this guy: https://github.com/aws/eks-anywhere/pull/652/files#diff-4438221d080eeaeb7e58673f2b24e69bf482eb366de82660ae94342614f9ef7dR106